src/components/CopyTextToClipboard.js

```diff
@@ -20,29 +20,21 @@ const defaultProps = {
     textStyles: [],
 };
 
-class CopyTextToClipboard extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.copyToClipboard = this.copyToClipboard.bind(this);
-    }
-
-    copyToClipboard() {
-        Clipboard.setString(this.props.text);
+const CopyTextToClipboard = (props) => {
+    const copyToClipboard = () => {
+        Clipboard.setString(props.text);
```

Comment:
@nhminhduc @PauloGasparSv Do you guys agree if we wrap this with `useCallback` since it has dependencies of `props.text`?